Libraries/Components/Button.windows.js

```diff
@@ -116,6 +147,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    let content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      const tabIndex = this.props.tabIndex || 0;
+      const windowsTabFocusable = !disabled && tabIndex >= 0;
+      content =
+        <FocusableWindows
+          ref = {(ref) => this._setFocusableRef(ref)}
```

Comment:
> =  [](start = 13, length = 3)

extra spaces

Reply:
k

---
In reply to: [157099327](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157099327) [](ancestors = 157099327)